internal/injector/aspect/advice/code/template.go

```diff
@@ -253,3 +254,30 @@ func numberLines(text string) string {
 
 	return strings.Join(lines, "\n")
 }
+
+func (t *Template) RenderHTML() string {
+	var buf strings.Builder
+
+	if len(t.imports) > 0 {
+		keys := make([]string, 0, len(t.imports))
+		for name := range t.imports {
+			keys = append(keys, name)
+		}
+		sort.Strings(keys)
+
+		buf.WriteString("\n\n```go\n")
+		buf.WriteString("// Assuming the following imports:\n")
```

Comment:
This formulation is confusing. My first read maybe me think that I had to manually import it to make orchestrion work